src/polyfills/CustomEvent.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
+(function () {
+
+  if ( typeof window.CustomEvent === "function" ) return false;
+
+  function CustomEvent ( event, params ) {
```

Comment:
I would prefer to just use https://github.com/kumarharsh/custom-event-polyfill since it seems more robust and then just change this to...

```
import 'custom-event-polyfill';
```

For example, this will now break all Node.js inclusions since it doesn't check for empty window object.